plugin/src/main/scala/org/apache/pekko/SearchIndex.scala

```diff
@@ -1,9 +1,4 @@
-/**
- * Copied and adopted from https://github.com/jonas/paradox-material-theme/blob/2d57fe0567ea9fe7e8de14faef4fa777841d505a/plugin/src/main/scala/io.github.jonas.paradox.material.theme/SearchIndex.scala
- *
- * Copyright (c) 2017-2018 Jonas Fonseca
- * License: MIT
```

Comment:
there is no license header on https://github.com/jonas/paradox-material-theme/blob/2d57fe0567ea9fe7e8de14faef4fa777841d505a/plugin/src/main/scala/io.github.jonas.paradox.material.theme/SearchIndex.scala

* we shouldn't make one up
* we have an MIT license for this content in our LICENSE file (includes copyright for Jonas Fonseca and Martin Donath)